### DISCUSSION TITLE
求助：3.9.5.81hook图片和voice消息

### DISCUSSION THREAD

**Discussion by cyonjan** [ttttupup/wxhelper#284] [General]
求助大侠，3.9.5.81以后不再有hook图片和voice等的消息，但是hookMsg可以侦听到type=3和37的消息，这时候怎样从xml中得到图片等的下载路径文件名。

**Comment by iamtryit** [comment]
我也想问这个问题。软件hook后接受端例子太简单，使用说明也不详细
